index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="./style.css">
+  <!--Favicon -->
+  <link rel="apple-touch-icon" sizes="180x180" href="./apple-touch-icon.png">
+  <link rel="icon" type="image/png" sizes="32x32" href="./favicon-32x32.png">
+  <link rel="icon" type="image/png" sizes="16x16" href="./favicon-16x16.png">
+  <link rel="manifest" href="./site.webmanifest">
+  <title>Project 4: Pizza bot &#127829</title>
+</head>
+
+<body>
+  <header>
+    <h1>WELCOME TO PIZZA BOT &#127829</h1>
+    <img class="header-img" src="pizza-header.jpg" alt="">
+    <h2>ENJOY PIZZA WHEREVER YOU ARE WHENEVER YOU WAN'T</h2>
+  </header>
+  <!--Pizza form -->
+  <section class="form">
+    <form class="pizza-order-form" id="form">
+      <h3>ORDER PIZZA HERE:</h3>
+      <br>
+      <br>
+      <label for="name">Write Your name</label>
+      <br>
+      <br>
+      <input name="name" id="name" type="text" placeholder="name" required>
+      <br>
+      <br>
+      <label for="pizzaSort">Choose pizzasort</label>
+      <br>
+      <br>
```

Comment:
Consider using `flexbox` or `padding`/`margin` instead of `<br>` to layout your elements

Reply:
Fixed! Added display flex and justify-content to the form. The height was set to 350px and width was set auto